.github/workflows/build_and_test.yml

```diff
@@ -72,32 +72,26 @@ jobs:
           flags: unittests
           fail_ci_if_error: true # optional (default = false)
 
-  build_and_test_with_code_coverage_win:
+  build_and_test_win:
     name: Build and test on Win
     runs-on: windows-latest
     steps:
       - uses: msys2/setup-msys2@v2
         with:
           msystem: MINGW64
           update: true
-          install: git mingw-w64-x86_64-toolchain
+          install: git mingw-w64-x86_64-toolchain mingw-w64-x86_64-curl
       - uses: actions/checkout@v2
       # ensure that gradle wrapper files in repository are valid by checking checksums
       - uses: gradle/wrapper-validation-action@v1
       - name: Set up JDK 11
         uses: actions/setup-java@v1
         with:
           java-version: 1.11
-
-      - name: Install curl Windows
-        continue-on-error: true
-        run: pacman -S --noconfirm mingw-w64-x86_64-curl
-        shell: msys2 {0}
-
       - name: Cache konan
         uses: actions/cache@v2
         with:
-          path: ~/.konan
+          path: 'C:\Users\runneradmin\.konan'
```

Comment:
There is no any env variable for this?

Reply:
I tried `HOME`, `HOMEPATH` and `USERPROFILE`, they don't seem to work, at least when configuring a step